examples/set-namespace-simple/README.md

```diff
@@ -1,10 +1,9 @@
-# set-namespace: Simple Example
+# set-namespace: Simple example
```

Comment:
nit: why change this? It will inconsistent with other examples.

Reply:
weird. GitHub seems not show the latest commit c115f5031c050c77db859820a88f43c25200ba85 which does not change this line and below (ln39) ?

Reply:
nevermind, it's an issue on my goland.